src/Chapter1.hs

```diff
@@ -615,8 +632,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 x = ones x + tens x
+    where 
+      ones :: Int -> Int 
+      ones n = mod (abs n) 10
+      tens :: Int -> Int 
+      tens n = mod (div (abs n) 10) 10
```

Comment:
Great one 👍🏼 

You can notice, that `lastDigit` from the previous task function can be reused in here 🙂 

On a side note, there is a standard function as well that combines both `div` and `mod` at once: `divMod`.

It returns the pair, so you can pattern match on that and get both variables at once:

```haskell
let (a, b) = abs n `divMod` 10
```